google-cloud-tools-plugin/ultimate/src/com/google/cloud/tools/intellij/appengine/server/instance/AppEngineRunConfigurationEditor.java

```diff
@@ -199,6 +184,11 @@ public void setAnchor(JComponent anchor) {
     this.anchor = anchor;
     myWebArtifactToDeployLabel.setAnchor(anchor);
     myPortLabel.setAnchor(anchor);
-    myServerParametersLabel.setAnchor(anchor);
+  }
+
+  // TODO(paflynn): Globally disable this checkstyle warning.
+  @SuppressWarnings("checkstyle:abbreviationaswordinname")
```

Comment:
I assumed this was there for some UI magic that Jetbrains does, but removing it doesn't seem to break anything.